src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+import { msg } from './intl'
+import './error_styles-nomodules.css'
+
+class GlobalErrorBoundary extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = { hasError: false }
+  }
+
+  componentDidCatch (error, info) {
+    this.setState({ hasError: true })
+    console.error(error)
+    console.error(info.componentStack)
+  }
+
+  render () {
+    const title = msg.globalErrorBoundaryTitle()
+    const descr = msg.globalErrorBoundaryDescription()
+    if (this.state.hasError) {
+      // You can render any custom fallback UI
+      return (
+        <div>
+          <nav className='navbar obrand_mastheadBackground obrand_topBorder navbar-pf-vertical'>
+            <div className='navbar-header'>
+              <a href='/' className='navbar-brand obrand_headerLogoLink' id={`pageheader-logo`}>
+                <img className='obrand_mastheadLogo' src={branding.resourcesUrls.clearGif} />
+              </a>
+            </div>
+          </nav>
+          <div className='container text-center globalErrorContainer' >
```

Comment:
For consistent in the end of tag must be no spaces, thus it must look like this:
`<div className='container text-center globalErrorContainer'>`